### DISCUSSION TITLE
Remove Trusted Account from bootstrap

### DISCUSSION THREAD

**Discussion by microblag** [aws/aws-cdk#22067] [Q&A]
I added an account "Primary" as a trusted account to "Sub". I now think I should have just set Sub as trust for lookup on Primary. When I try and re-run bootstrap in Sub without the trust argument nothing changes. Is it safe to remove an account from TrustedAccounts (I have deployed using this trust relationship) and if so, how?

**Comment by peterwoodworth** [comment]
Hey, sorry no one was able to get to this until you've opened your issue. I've left a comment [here](https://github.com/aws/aws-cdk/issues/22703#issuecomment-1298009080), we should use the thread I've just linked to to continue this discussion as we should adjust documentation at the very least. Thanks!
